test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,111 @@
+./test_scripts/API/SetAudioStreamingIndicator/045_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_TIMED_OUT.lua Check_that_SDL_responds_with_resultcode_TIMED_OUT_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/046_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_CHAR_LIMIT_EXCEEDED.lua Check_that_SDL_responds_with_resultcode_CHAR_LIMIT_EXCEEDED_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/047_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_INVALID_ID.lua Check_that_SDL_responds_with_resultcode_INVALID_ID_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/048_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_DUPLICATE_NAME.lua Check_that_SDL_responds_with_resultcode_DUPLICATE_NAME_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/049_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_replies_TRUNCATED_DATA.lua Check_that_SDL_responds_with_resultcode_TRUNCATED_DATA_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/050_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_WARNINGS.lua Check_that_SDL_responds_with_resultcode_WARNINGS_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/051_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_WRONG_LANGUAGE.lua Check_that_SDL_responds_with_resultcode_WRONG_LANGUAGE_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/052_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_RETRY.lua Check_that_SDL_responds_with_resultcode_RETRY_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/053_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_SAVED.lua Check_that_SDL_responds_with_resultcode_SAVED_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/054_ATF_P_TC_SetAudioStreamingIndicator_media_app_HMI_replies_UNSUPPORTED_RESOURCE.lua Check_that_SDL_responds_with_resultcode_UNSUPPORTED_RESOURCE_from_HMI_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_doesnot_reply_GENERIC_ERROR.lua Check_that_SDL_responds_with_resultcode_GENERIC_ERROR_HMI_doesnot_reply_incase_SetAudioStreamingIndicator_is_correct
+./test_scripts/API/SetAudioStreamingIndicator/056_ATF_N_TC_SetAudioStreamingIndicator_GENERIC_ERROR_HMI_wrongtype_audioStreamingIndicator.lua Check_that_SDL_responds_GENERIC_ERROR_when_HMI_replies_wrong_type_audioStreamingIndicator
```

Comment:
@dboltovskyi, this script checks that SDL invalidates response from HMI and returns GENERIC_ERROR .
Test 013 checks that SDL invalidates request from mobile.